### DISCUSSION TITLE
Filter on Operational Status Column

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#74] [Shipped - Live on production]
![image](https://github.com/Samedis-care/samedis-care-community/assets/117633901/b1bf884f-cdfd-4aaa-ada4-bdcc59f0efbc)

Can a filter be added to the operation status column? So I can multiselect all statuses except Retired (as I never want to see those normally). Thanks!

**Comment by YvesR** [comment]
You want this? Coming in next update.

![image 2023-08-23 um 07 43 04](https://github.com/Samedis-care/samedis-care-community/assets/4928882/25b0adff-b0b5-490f-8255-e872625da07e)

**Reply by BioMedGPH** [reply]
This is current implementation right? Implementation will be a column filter like this i assume:
![image](https://github.com/Samedis-care/samedis-care-community/assets/117633901/d7693463-aad4-4e30-abdd-e2c9f845f3d7)

**Reply by BioMedGPH** [reply]
would be great, so i can have all items except retired in a view

**Reply by YvesR** [reply]
we added another option to show all except retired with the operation filter, this is not a column filter.
